website/content/docs/k8s/observability/metrics.mdx

```diff
@@ -76,4 +74,21 @@ global:
 
 ## Metrics in the UI Topology Visualization
 
+Consul’s built in UI has a topology visualization for services part of the Consul-Connect service mesh. The topology visualization has the ability to fetch basic metrics from a metrics provider for each service and display those metrics as part of the topology visualization.
```

Comment:
Can you link to the topology visualization docs here?

Reply:
✅